src/pages/chart.jsx

```diff
@@ -0,0 +1,22 @@
+import EmbedPlayer from "../components/embedPlayer";
+
+const domain = process.env.NEXT_PUBLIC_EMBED_DOMAIN_URL;
+
+export async function getServerSideProps(context) {
+  const { query } = context;
+  const queryString = Object.keys(query)
+    .map((key) => `${key}=${query[key]}`)
+    .join("&");
```

Comment:
alt way to do this, though I don't think it's better or worth changing.
```suggestion
  const queryString = Object.entries(query)
    .map((entry) => entry.join("="))
    .join("&");
```